TraceScreen.c

```
@@ -60,6 +60,16 @@ void TraceScreen_delete(Object* cast) {
    free(InfoScreen_done((InfoScreen*)this));
 }
 
+static void TraceScreen_scan(InfoScreen* super) {
+   TraceScreen* this = (TraceScreen*) super;
+   if (!this->strace_alive) {
+      char* err = NULL;
```

Cppcheck (2.21.0)

[style] The function 'HeaderLayout_getName' is never used.

(unusedFunction)